lambdas/handlers/document_manifest_by_nhs_number_handler.py

```diff
@@ -2,49 +2,61 @@
 import os
 
 from botocore.exceptions import ClientError
-from enums.metadata_field_names import DocumentReferenceMetadataFields
-from models.document import Document
+from enums.supported_document_types import SupportedDocumentTypes
+from lambdas.utils.decorators.validate_patient_id import validate_patient_id
+from lambdas.utils.decorators.ensure_env_var import ensure_environment_variables
+from services.manifest_dynamo_service import ManifestDynamoService
 from services.document_manifest_service import DocumentManifestService
-from services.dynamo_service import DynamoDBService
-from utils.exceptions import (DynamoDbException, InvalidResourceIdException,
+from utils.decorators.validate_document_type import validate_document_type
+from utils.exceptions import (DynamoDbException,
                               ManifestDownloadException)
 from utils.lambda_response import ApiGatewayResponse
-from utils.utilities import validate_id
 
 logger = logging.getLogger()
 logger.setLevel(logging.INFO)
 
 
+@validate_patient_id
+@validate_document_type
+@ensure_environment_variables(
+    names=["DOCUMENT_STORE_DYNAMODB_NAME",
+           "LLOYD_GEORGE_DYNAMODB_NAME",
+           "ZIPPED_STORE_BUCKET_NAME",
+           "ZIPPED_STORE_DYNAMODB_NAME"
+           ]
+)
 def lambda_handler(event, context):
     logger.info("Starting document manifest process")
 
     try:
         nhs_number = event["queryStringParameters"]["patientId"]
-        validate_id(nhs_number)
+        doc_type = event["queryStringParameters"]["docType"]
 
-        document_store_table_name = os.environ["DOCUMENT_STORE_DYNAMODB_NAME"]
-        lloyd_george_table_name = os.environ["LLOYD_GEORGE_DYNAMODB_NAME"]
         zip_output_bucket = os.environ["ZIPPED_STORE_BUCKET_NAME"]
         zip_trace_table_name = os.environ["ZIPPED_STORE_DYNAMODB_NAME"]
         # zip_trace_ttl = os.environ["DOCUMENT_ZIP_TRACE_TTL_IN_DAYS"]
 
-        dynamo_service = DynamoDBService()
+        dynamo_service = ManifestDynamoService()
+        ds_documents = []
```

Comment:
What does ds documents stand for? can we rename this to arf?

Reply:
This was just existing code. Personally I think it should be EHR, but it's written as ARF everywhere else. I believe it's 'Document Store'

Either way, this will be changed to ARF in the next commit